Index: cim-apidoc/web/cim-api.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>windows-1252
===================================================================
--- cim-apidoc/web/cim-api.json	(revision ad0acb28a90b7ecb8b64c6fccd20974fd367098a)
+++ cim-apidoc/web/cim-api.json	(revision )
@@ -1,11 +1,11 @@
 {
     "swagger": "2.0",
     "info": {
-        "description": "API documentation for the Endeavour Health Common Interface Mechanism.<div id='notes'><div id='notestitle'>NOTES</div><p>Sample data and notes on usage can be <a href='usage-notes.html' target='_blank'>viewed here</a>.</p><p>Build last updated: 17 August 2015 19:15</p></div>",
+        "description": "API documentation for the Endeavour Health Common Interface Mechanism.<div id='notes'><div id='notestitle'>NOTES</div><p>Sample data and notes on usage can be <a href='usage-notes.html' target='_blank'>viewed here</a>.</p><p>Build last updated: 17 August 2015 11:50</p></div>",
         "version": "0.1",
         "title": "Common Interface Mechanism"
     },
-    "host": "localhost:8080",
+    "host": "endeavour-cim.cloudapp.net",
     "basePath": "/api/v0.1",
     "tags": [
         {
\ No newline at end of file
Index: cim-dataservice/src/main/java/org/endeavourhealth/cim/Registry.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>windows-1252
===================================================================
--- cim-dataservice/src/main/java/org/endeavourhealth/cim/Registry.java	(revision ad0acb28a90b7ecb8b64c6fccd20974fd367098a)
+++ cim-dataservice/src/main/java/org/endeavourhealth/cim/Registry.java	(revision )
@@ -8,11 +8,11 @@
     // Items that need changing based on deployment
     ////////////////////////////////////////////////////////////
 
-        private static final String EMIS_SOAP_URI = "http://localhost:9001/GPApiService/Soap";
-        // private static final String EMIS_SOAP_URI = "http://endeavour-gp.cloudapp.net:9001/GPApiService/Soap";
+        // private static final String EMIS_SOAP_URI = "http://localhost:9001/GPApiService/Soap";
+        private static final String EMIS_SOAP_URI = "http://endeavour-gp.cloudapp.net:9001/GPApiService/Soap";
 
-        private static final String BASE_URI = "http://localhost:8080/api/0.1";
-        // private static final String BASE_URI = "http://endeavour-cim.cloudapp.net/api/0.1";
+        // private static final String BASE_URI = "http://localhost:8080/api/0.1";
+        private static final String BASE_URI = "http://endeavour-cim.cloudapp.net/api/0.1";
 
 
         // ALSO CHANGE cim-apidoc/web/cim-api.json
